### ISSUE TITLE
Carousel text background fails automated accessibility checks

### ISSUE THREAD

**Issue by timdavison** [misd-service-development/drupal-cambridge-theme#123]
Wave fails our caption text with a contrast error.
`Very Low Contrast: <span class="campl-slide-caption-txt">Subscribe to the HR Bulletin</span>`

The background is set by a parent element which is positioned absolutely. It's a black transparent png.
background: url(/sites/all/themes/cambridge_theme/images/interface/bg-carousel-caption.png) repeat 0 0;

Whilst I think this is actually ok from an accessibility point of view, it be be simpler to just set a background colour with an opacity, and automated checkers would find this easier to deal with. e.g.
`.campl-slide-caption {
  background:rgba(0,0,0,0.8);
}`
